src/Components/Atoms/FAAvatar/index.tsx

```diff
@@ -0,0 +1,11 @@
+import React, { memo } from "react";
+import { Avatar } from "@mui/material";
+import { IFAAvatarProps } from "./FAAvatar.interface";
```

Comment:
suggestion to add one empty line between import order

```suggestion
import { Avatar } from "@mui/material";

import { IFAAvatarProps } from "./FAAvatar.interface";
```